ci/pipelines/ubuntu-noble/pipeline.yml

```
@@ -73,8 +73,9 @@ jobs:
         - name: bosh-linux-stemcell-builder
     - task: build-docker-args
       file: bosh-stemcells-ci/ci/tasks/build-docker-args.yml
-      image: os-image-stemcell-builder
-    - put: os-image-stemcell-builder
+      image: os-image-stemcell-builder-registry-image #! TODO: image should be stock ubuntu
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)